deepeval/test_run/api.py

```diff
@@ -1,14 +1,36 @@
 from pydantic import BaseModel, Field
 from typing import Optional, List
-
+from deepeval.test_case import LLMTestCaseParams
+from deepeval.types import Languages
+from langchain_core.embeddings import Embeddings
 
 class MetricsMetadata(BaseModel):
+    strict_mode: bool
     metric: str
     score: float
     threshold: float
     success: bool
+    evaluation_model: Optional[str] = None
     reason: Optional[str] = None
-    evaluation_model: Optional[str] = Field(None, alias="evaluationModel")
+
+    # Optional attributes (Not every BaseMetric has these attributes)
```

Comment:
I wouldn't add metrics configurations such as `criteria`, `n`, `include_reason` here because `MetricsMetadata` maps 1:1 to the schema on Confident.